src/main/java/org/fcrepo/spec/testsuite/crud/HttpPut.java

```diff
@@ -90,7 +113,7 @@ public void httpPut(final String uri) {
      */
     @Test(groups = {"MUST"})
     @Parameters({"param1"})
-    public void httpPutpdateTriples(final String uri) {
+    public void httpPutUpdateTriples(final String uri) {
```

Comment:
Line 117:  "httpPutpdateTriples" -> "httpPutUpdateTriples"

Reply:
That is the change I made, it was a type before and now it is `httpPutUpdateTriples`

Reply:
I generally get the impression this test suffered at the hand of multiple IDEs / merge issues, given the missing test, some of the formatting issues and this typo

Reply:
Ah sorry misinterpreted this (waking up needed), i have updated line 117